docs/source/nodes_and_pipelines/run_a_pipeline.md

```diff
@@ -110,6 +113,10 @@ class DryRunner(AbstractRunner):
         )
         self._logger.info("Checking inputs...")
         input_names = pipeline.inputs()
+
+        # Register the default dataset pattern with the catalog
+        catalog = catalog.shallow_copy(extra_dataset_patterns=self._extra_dataset_patterns)
```

Comment:
This is already done at the abstract runner level right? Why is it necessary to do it again in `_run()`?